nilearn/glm/first_level/design_matrix.py

```diff
@@ -298,9 +299,11 @@ def make_first_level_design_matrix(
     frame_times : array of shape (n_frames,)
         The timing of acquisition of the scans in seconds.
 
-    events : :obj:`pandas.DataFrame` instance or None, default=None
+    events : :obj:`pandas.DataFrame` instance, \
+             or :obj:`str` or :obj:`pathlib.Path` to a CSV or TSV file, \
+             or None, default=None
```

Comment:
This is not clear to me here if we expect a single entity or a list. If we accept also list and tuple (the case verified below), we could mention it.

When events is specified as a list, we only take the first DataFrame returned from `check_and_load_tables` . I think this could also be mentioned. 

Or we could decide not to accept a list or tuple and raise error in that case.

Reply:
went with not accepting a list